### PR TITLE
feat: Improve examples and enable playground to work on dev mode 

### PR DESCRIPTION
**Pull request by nimaa77**
## Summary

this pull request makes the deno examples usable, each example needs users to provide an APIKey and possibly some other parameter like accountID, so deno scripts now ask users to enter APIKey and other values and later those values are used when SDK functions get called.

also, the playground needs to push a config object to the SDK config based on the deployment environment, if we are using the app in dev mode the SDK should point to the development instance which requires the playground to inject the URLs to the SDK config

```typescript
const sdkConfig: Config = {
  exchange: {
    serviceUrl: "https://dev-deployment.tradrapibackend.com",
    socketUrl: "wss://dev-deployment.tradrapibackend.com"
  },
  auth: { apiKey: "<API-KEY>" },
  debug: true,
};
```

* example file name changed so they match the code (x.g: list accounts was fetching a single account detail)
* New examples  🔥
* updated README.md file, now it acts as documentation to guide devs add/update new examples 

Example

### Deno 

The `prompt` function takes the parameters needed to execute the examples

```typescript
const apiKey = prompt("Enter your API key:");

const accountID = prompt("Enter the account ID to retrieve:");
```

users can execute the script using a bash script

```bash
deno run  --allow-read --allow-net --allow-env https://raw.githubusercontent.com/nimaa77/examples/nima/x-261-playground-examples/deno/account-balance.ts
```

### Playground

so to push the `exchange object` into the config we have two options:
1. use regex to find a label and replace it with the request value
2. parse the AST find the related object manipulate the tree and then pass it to the run time

I chose the first solution for simplicity, so the playground will look for `// <EXCHANGE-CONFIG>` and replace it with the exchange object, or remove the label

```typescript
// configure the SDK
const sdkConfig: Config = {
  // <EXCHANGE-CONFIG>
  auth: { apiKey: "<API-KEY>" },
  debug: true,
};
```

### CODE REVIEW

File: playground/examples/account-balance.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// retrieve trading account balance, equity, and margin information
+
+import { Tradr, Config } from "@tradrapi/trading-sdk";
+
+// configure the SDK
+const sdkConfig: Config = {
+  // <EXCHANGE-CONFIG>
```
Comment (GabrielGil) [inline playground/examples/account-balance.ts:7]: It is actually not a exchange. It should probably be simply "SDK Config", IMO.

File: playground/examples/list-accounts.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// create an instance of the SDK
+const tradrApi = Tradr.make(sdkConfig);
+
+// filter and sort the accounts based on your needs
+const filters: ListAccountDto = {
+  // sort the accounts by creation date
+  sortBy: "createdAt",
+  // number of accounts to retrieve per request
+  limit: 10,
+  // ℹ️ ⬇ ️apply more filters, press ctrl+space to see all available filters
```
Comment (GabrielGil) [inline playground/examples/list-accounts.ts:21]: Nice detail ;)

File: playground/examples/trading-bot.ts
Comment (GabrielGil) [inline playground/examples/trading-bot.ts:1]: Nice thing that will actually be impressive and work even without specifying an account ID.

**Review by GabrielGil — APPROVED** [review]
By me, it is fine. Kindly check with @andrewbrg too :)

**Review by andrewbrg — APPROVED** [review]

**Pull request merged**